### DISCUSSION TITLE
blog/2024/09/28/%E5%94%90%E5%AE%8C%E4%BA%86/

### DISCUSSION THREAD

**Discussion by giscus** [ZnPdCo/blog#4] [Announcements]
# blog/2024/09/28/%E5%94%90%E5%AE%8C%E4%BA%86/

https://znpdco.github.io/blog/2024/09/28/%E5%94%90%E5%AE%8C%E4%BA%86/

**Comment by ZnPdCo** [comment]
今天cf看错题，B题卡了1个小时，唐完了：

![屏幕截图 2024-10-06 153403](https://github.com/user-attachments/assets/f42b1dc2-a28d-45ab-ab95-261c27742787)

**Comment by ZnPdCo** [comment]
唐完了

**Comment by ZnPdCo** [comment]
做多少遍就唐多少遍

**Comment by ZnPdCo** [comment]
今天hjy终于知道 $\sum_{i=1}^m v_i=n$ 时 $\sum_{i=1}^m v_i^2\le n^2$ 了
